scripts/ProcessTemplates.ps1

```diff
@@ -1,24 +1,50 @@
 
 $mainPath = Split-Path (split-path -parent $MyInvocation.MyCommand.Path) -Parent
 
-$reports = Get-ChildItem $mainPath
 $reportTypes = @('Cohorts', 'Workbooks')
 $templateExtensions = @('cohort', 'workbook')
 $defaultLanguage = 'en-us'
 $payload = @{ }
 $categoryMetadataFileName = 'categoryResources.json'
 
 # This is the name of where the blob that ALM looks for
-$azureBlobFilename = "community-templates-V2.json";
-
+$azureBlobFileNameBase = "community-templates-V2";
+
+$docRepos = @(
+    @{ repo="Application-Insights-Workbooks.cs-cz"; lang="cs-cz"}, 

```

Comment:
do you need the language of the repo separate from the name of the repo?  it seems like passing this info 2x is a super way to be wrong?  instead would it make more sense to remove it from the name of the repo here, and just append it in the code when `lang !== "en-us"` ?

Reply:
simplify the languages and use just the lang